easybuild/easyblocks/c/clang.py

```diff
@@ -491,22 +514,54 @@ def sanity_check_step(self):
         if LooseVersion(self.version) >= LooseVersion('3.8'):
             custom_paths['files'].extend(["lib/libomp.%s" % shlib_ext, "lib/clang/%s/include/omp.h" % self.version])
 
+        # Detect OpenMP support for CPU architecture
+        arch = get_cpu_architecture()
+        # Check architecture explicitly since Clang uses potentially
+        # different names
+        if arch == X86_64:
+            arch = 'x86_64'
+        elif arch == POWER:
+            arch = 'ppc64'
+        elif arch == AARCH64:
+            arch = 'aarch64'
+        else:
+            print_warning("Unknown CPU architecture (%s) for OpenMP library check!" % arch)
+        custom_paths['files'].extend(["lib/libomptarget.%s" % shlib_ext,
+                                      "lib/libomptarget.rtl.%s.%s" % (arch, shlib_ext)])
+        # If building for CUDA check that OpenMP target library was created
```

Comment:
This should have been version restricted as it is causing sanity checks to fail with `Clang-11.0.1-GCCcore-10.2.0.eb`

Reply:
I said that because of https://gist.github.com/boegelbot/952bd8e34ed36a074a6e1fb7ad4f0ad9 but waiting on other test reports to confirm

Reply:
@ocaisa Worked fine with `Clang-11.0.1-gcccuda-2020b.eb` though in my test report?